src/Event/AfterApplicationInitializationEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Castor\Event;
+
+use Castor\Console\Application;
+use Castor\TaskDescriptor;
+
+class AfterApplicationInitializationEvent
+{
+    public function __construct(
+        private readonly Application $application,
```

Comment:
Didn't know if really required to pass the application here, because already available in `GlobalHelper::getApplication()` or `app` function

Better to keep or remove ?

Reply:
If the application state realistically cannot differ between the one passed as a parameter and the global state accessed at the start of the function, then it may be advisable to remove the parameter.

However, I'm considering whether task parallelization could invalidate this assumption. Could a task running in parallel theoretically alter the application state between the moment an event is dispatched and when it is listened to?

Reply:
I'm not sure, but I don't think so. Except in a case of parallel maybe?